arangod/Agency/Supervision.cpp

```diff
@@ -1010,6 +1011,101 @@ void Supervision::workJobs() {
 }
 
 
+void Supervision::readyOrphanedIndexCreations() {
```

Comment:
This could have been made a static method such that unittests are possible.

Reply:
good,